src/Tools/dotnet-user-jwts/test/UserJwtsTests.cs

```diff
@@ -80,6 +71,22 @@ public void Create_WritesGeneratedTokenToDisk()
         Assert.Contains("dotnet-user-jwts", File.ReadAllText(appsettings));
     }
 
+    [Fact]
+    public void Create_CanModifyExistingScheme()
+    {
+        var project = Path.Combine(_fixture.CreateProject(), "TestProject.csproj");
+        var appsettings = Path.Combine(Path.GetDirectoryName(project), "appsettings.Development.json");
+        var app = new Program(_console);
+
+        app.Run(new[] { "create", "--project", project });
+        Assert.Contains("New JWT saved", _console.GetOutput());
+        var exception = Record.Exception(() => JsonSerializer.Deserialize<Dictionary<string, Jwt>>(File.ReadAllText(appsettings)));
```

Comment:
I'm sure this follows an existing pattern, but why not just let it throw?

Reply:
We are trying to verify that it does not throw. This test validates that the previous behavior, where in calling the CLI command twice would write invalid JSON to the file, now longer occurs. Similarly, this is why this test case doesn't validate the actual contents of the file since there are other tests that cover that.

Reply:
Wouldn't the test fail if that call threw?  Is that different from it failing on the next line when we assert that the exception is null?

> Similarly, this is why this test case doesn't validate the actual contents of the file since there are other tests that cover that.

I'm not sure I follow.  There's another test that updates the scheme and validates that the new one is in place?  If so, why have this test at all?  If not, isn't it worthwhile to confirm that the update actually took effect?

Reply:
> Wouldn't the test fail if that call threw? Is that different from it failing on the next line when we assert that the exception is null?

Yes, the test would fail if the call threw but this is a more explicit way of clarifying aht we don't expect it to through.

> There's another test that updates the scheme and validates that the new one is in place? If so, why have this test at all? If not, isn't it worthwhile to confirm that the update actually took effect?

I double checked and it looks like we don't have a test that confirms these updates yet so I've updated the code change to match it appropriately.